Classes/Service/ClassBuilder.php

```diff
@@ -584,7 +584,7 @@ protected function buildActionMethod(Model\DomainObject\Action $action, Model\Do
             $actionMethod->updateParamTags();
 
             if ($actionName === 'edit') {
-                $actionMethod->setTag('ignorevalidation', '$' . $parameterName);
+                $actionMethod->setTag('TYPO3\CMS\Extbase\Annotation\IgnoreValidation' . '("' . $parameterName . '")');
```

Comment:
And one more nitpick:
`'TYPO3\CMS\Extbase\Annotation\IgnoreValidation' . '("' . $parameterName . '")'`
should be
`'TYPO3\CMS\Extbase\Annotation\IgnoreValidation("' . $parameterName . '")'`